src/components/UserProfile/components/UserAcquiredSkills/UserEditableSkills.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { gql } from "apollo-boost";
+import ChosenSkills from "../ChosenSkills";
+import SkillsPicker from "../../../SkillsPicker";
+import EditButton from "../../../common/EditButton";
+
+const userAddSkills = gql`
+  mutation userAddSkills ($skill_ids:[ID!]!) {
+    userAddSkills(skill_ids:$skill_ids) {
+      id
+      acquired_skills {
+        id
+        name
+        category
+        showcase_count
+      }
+      requested_skills {
+        id
+        name
+      }
+    }
+  }
+`;
+
+class UserEditableSkills extends React.Component {
+  state = {
+    isEditing: false, // used to control rendering of SkillsPicker
+    shouldSave: false, // used to control form submit in SkillsPicker
+  }
+
+  toggleEdit = () => this.setState(
+    { isEditing: !this.state.isEditing }, // toggle editing mode
+    () => this.state.shouldSave && this.toggleSave(), // reset shouldSave when closing
+  );
+
+  toggleSave = () => this.setState(
+    { shouldSave: !this.state.shouldSave },
```

Comment:
<img width="502" alt="screenshot 2018-10-19 at 21 20 37" src="https://user-images.githubusercontent.com/6613473/47239239-06366600-d3e5-11e8-91fa-164117bda5a2.png">
Trying to figure out why `.toggleSave` is firing twice. 
It also errors on empty input submit, but only the first time